src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -588,6 +588,21 @@
     AnalyzeRequestBuilder prepareAnalyze(String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithIndexAndMultiValued(@Nullable String index, String... text);
+
+    /**
+     * Analyze texts.
+     *
+     * @param text The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithMultiValued(String... text);
```

Comment:
Oh, sorry, and I completely agree about the naming convention.
Other question.
If we can change I/F about prepareAnalyze, what do you think about dropping all text parameters for I/F simplification and adding the setter of text/texts to AnalyzeRequestBuilder?

Of course, we are hard to find the difference between prepareAnalyze(String text), that is old version, and prepareAnalyze(String index), that is new one.